include/fi_atom.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
 
 
 #if ENABLE_DEBUG
-#define ATOMIC_DEF_INIT int is_initialized
+#define ATOMIC_DEF_INIT int is_initialized;
```

Comment:
It is very unusual for a define to have the ';' as part of the definition.  Is there some other way we can eliminate this, possibly by removing the typedef struct ofi_atomic<radix>_t in the non-debug case?  (I'm not sure if it's possible given how the macros are defined).